click_odoo_contrib/backupdb.py

```diff
@@ -59,7 +59,7 @@ def _backup_filestore(dbname, backup):
 )
 @click.option(
     "--format",
-    type=click.Choice(["zip", "folder"]),
+    type=click.Choice(["zip", "dump", "folder"]),
     default="zip",
     show_default=True,
     help="Output format",
```

Comment:
While you are at it, it would be cool to expand the help to explain the meaning of each format.